js/join/main.js

```diff
@@ -166,6 +166,20 @@ const previewManager = new       PreviewManager(byEID("session-preview-image"));
 const statusManager  = new     AppStatusManager(byEID("status-value"));
 
 document.addEventListener("DOMContentLoaded", () => {
+  const closedChatBox = document.getElementById("chat-box-closed");
+  const openChatBox = document.getElementById("chat-box-open");
+  const openChatHeader = document.getElementById("open-chat-header");
+
+  closedChatBox.onclick = () => {
+    openChatBox.style.display = "block";
+    closedChatBox.style.display = "none";
+  };
+
+  openChatHeader.onclick = () => {
+    openChatBox.style.display = "none";
+    closedChatBox.style.display = "flex";
+  };
+
```

Comment:
Not sure where to put this comment, but I'd like to see a change made to the chat box where, if we have unread messages (other than "census" messages), the chat box shows something to indicate this (preferably, the number of unread messages).  This will cue people to actually expand chat and check it out.

Reply:
> Is there any easy way for me to tell / detect if we have unread messages? [...] If not, how might I go about this?

Nope, not at all.  You'd need to write the code for this.

I'd like to see this PR merged by Thursday, so maybe we should delay this until after merging the rest of the PR, because I don't think this specific change is really likely to be ready for Friday.  I can make a separate issue for this.

But the relevant code will involve [`ChatManager`](https://github.com/NetLogo/HubNet-Web/blob/4764ec91b9af9c7fefc3e92f20242ea1db7f2860/js/common/ui/chat-manager.js) in some way.

Reply:
Read this, but will implement in the next PR (discussed together).